chpl/chpl-resources/src/main/resources/errors.properties

```diff
@@ -494,3 +494,12 @@ filter.notPresent=No filter object was present to be saved.
 filter.empty=Filter cannot be empty.
 filter.noUser=The filter does not have an associated user.
 filter.noFilterType=The filter does not have an associated filter type.
+
+complaints.create.statusOpen=When creating a new complaint, the Complaint Status must be open.
```

Comment:
Is this accurate?

Reply:
![image](https://user-images.githubusercontent.com/24817202/59053427-6d633680-885f-11e9-89eb-d8d1fea78f1c.png)

Reply:
Should the status be not null in the data model then?